ProcessLib/InitialCondition.h

```diff
@@ -70,7 +70,8 @@ class MeshPropertyInitialCondition : public InitialCondition
 		assert(_property.getMeshItemType() == MeshLib::MeshItemType::Node);
 	}
 
-	virtual double getValue(MeshLib::Node const& n) const override
+	virtual double getValue(MeshLib::Node const& n,
+	                        int const component_id) const override
```

Comment:
Getting values in process variable (see [here](ffbb9ac68f450ebef691cc97b4d0d2b6ec81e176)) for each component when called from [Process::setInitialConditions()](https://github.com/endJunction/ogs/blob/MulticomponentProcessVars/ProcessLib/Process.h#L183).